analyzer/codechecker_analyzer/cmd/analyze.py

```diff
@@ -102,6 +102,16 @@ def add_arguments_to_parser(parser):
                              "specified file rather than invoke the compiler "
                              "executable.")
 
+    parser.add_argument('--keep-gcc-fix-include',
+                        dest="keep_gcc_fix_include",
+                        required=False,
+                        action='store_true',
+                        default=False,
```

Comment:
Shouldn't be the default `True` to keep them?

Reply:
I chose False because we still don't know what was the reason of omitting these paths. Only one user reported an error about a missing `include-fixed` directory so far.